### PR TITLE
TEA stats epub fixes

### PR DESCRIPTION
**Pull request by tomjw64**
EPUB will not build for stats without these changes

### CODE REVIEW

**Comment by tomjw64** [comment]
This does not address any issues. Do not assign to QA.

File: rulesets/books/statistics/_config.scss [hunk 1]
```diff
@@ -178,7 +178,6 @@ $Config_TargetLabels: append($Config_TargetLabels,
 */
 $Config_Notes: (
   (className: 'calculator', moveSolutionTo: $AREA_NONE,  labelText: "Using the TI-83, 83+, 84, 84+ Calculator"),
-  (className: 'finger',     moveSolutionTo: $AREA_NONE,  labelText: "Note", replaceHeader: true),
```
Comment (philschatz) [inline rulesets/books/statistics/_config.scss]: Was this in a HTML snippet?

Comment (tomjw64) [inline rulesets/books/statistics/_config.scss]: Ah, yes. I should remove the snippet.

Comment (philschatz) [inline rulesets/books/statistics/_config.scss]: awww yeah! And travis caught there was a dangling CSS comment that referred to the snippet 😄

**Pull request merged**